core/src/main/java/com/sequenceiq/cloudbreak/service/recipe/RecipeService.java

```diff
@@ -116,6 +116,23 @@ public Recipe createForLoggedInUser(Recipe recipe, @Nonnull Long workspaceId, St
         }
     }
 
+    public Recipe createWithInternalUser(Recipe recipe, @Nonnull Long workspaceId, String accountId) {
+        if (recipeViewRepository.findByNameAndWorkspaceId(recipe.getName(), workspaceId).isPresent()) {
```

Comment:
Can we somehow differentiate recipes that are created by our application or by a regular user?
Simply thinking if some day we wish hide these, can we spare a problematic migration script at this point?

Question stands for all internally created resource.

Reply:
Good question. I can think of an extra column like `applicationCreated` or `internal`, that we will set true when it was created through an internal API. @lnardai @horadla23 what do you think about this solution?

Reply:
I would prefer an enum for that (in case if we need something more in the future), like CreationType, with values USER and SERVICE, but anyway I like the idea